rust/src/data_structures.rs

```diff
@@ -76,14 +77,14 @@ pub struct ListenInfo {
 }
 
 impl ListenInfo {
-    pub(crate) fn to_fbs(self) -> transport::ListenInfo {
+    pub(crate) fn to_fbs(&self) -> transport::ListenInfo {
```

Comment:
Rust told me to change this to avoid confusing naming since `self` is passed as reference after removing `Copy` derive (or something like that).